package.json

```diff
@@ -1,33 +1,19 @@
 {
   "name": "@behance/eref-js",
+  "version": "2.0.2",
   "main": "src/main",
+  "license": "MIT",
   "description": "Eref handling for Livefyre arc4 encoded messages",
   "author": "Livefyre <support@livefyre.com>",
   "repository": "https://github.com/Livefyre/eref-js",
   "contributors": {
     "name": "Ross Pfahler",
     "email": "ross@livefyre.com"
   },
-  "license": "MIT",
-  "version": "2.0.2",
-  "dependencies": {
-    "inherits": "Livefyre/inherits.git#ccbfcef",
```

Comment:
Only important change is this (remove 2 deps for src/erefprocessor.js) which isn't used